ecs-cli/modules/commands/local/local_command.go

```diff
@@ -48,12 +48,12 @@ func createCommand() cli.Command {
 	}
 }
 
-// TODO This is a placeholder function used to test the ECS local network configuration.
 func upCommand() cli.Command {
 	return cli.Command{
 		Name:   "up",
 		Usage:  "Create a Compose file from an ECS task definition and run it.",
```

Comment:
Is that kind of too much overlap compared to `createCommand`?  Except for running it.

Reply:
Yes but that's expected. We split `create` and `up` because it's possible that a user wants to modify the Compose file after running `create` and then they can just run `up`.

I just created a backlog task to change the behavior of `local create` slightly: https://github.com/aws/amazon-ecs-cli/issues/809